Nightly integration tests broken

The nightly integration tests are failing to configure.  I think this is probably a result of upgrading the test image here https://github.com/llvm/circt/commit/d2ff58a59cdcb0e925c47fe76c6afe3505e7c034.

```
CMake Error at cmake/modules/HandleLLVMOptions.cmake:295 (message):
  Host compiler does not support '-fuse-ld=lld'
Call Stack (most recent call first):
  CMakeLists.txt:737 (include)


-- Configuring incomplete, errors occurred!
```

The short integration tests are not broken, but it only uses `clang` and not `gcc`.

--- a/circt-ci-build/Dockerfile
+++ b/circt-ci-build/Dockerfile
@@ -26,4 +26,5 @@ RUN ln -s /usr/bin/clang-13 /usr/bin/clang;\
     ln -s /usr/bin/clang-format-13 /usr/bin/clang-format;\
     ln -s /usr/bin/clang-format-diff-13 /usr/bin/clang-format-diff;\
     ln -s /usr/bin/git-clang-format-13 /usr/bin/git-clang-format;\
-    ln -s /usr/bin/lld-13 /usr/bin/lld
+    ln -s /usr/bin/lld-13 /usr/bin/lld;\
+    ln -s /usr/bin/lld-13 /usr/bin/ld.lld

--- a/integration_test/Dockerfile
+++ b/integration_test/Dockerfile
@@ -30,7 +30,8 @@ RUN ln -s /usr/bin/clang-13 /usr/bin/clang;\
     ln -s /usr/bin/clang-format-13 /usr/bin/clang-format;\
     ln -s /usr/bin/clang-format-diff-13 /usr/bin/clang-format-diff;\
     ln -s /usr/bin/git-clang-format-13 /usr/bin/git-clang-format;\
-    ln -s /usr/bin/lld-13 /usr/bin/lld
+    ln -s /usr/bin/lld-13 /usr/bin/lld;\
+    ln -s /usr/bin/lld-13 /usr/bin/ld.lld
 
 COPY *.sh /tmp/
 

